var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -115,14 +131,23 @@ def install(self, spec, prefix):
                 'METIS_OPT=-I%s' % spec['metis'].prefix.include,
                 'METIS_LIB=%s' % metis_lib])
 
-        if '+mpi' in spec:
+        if 'mpi' in spec:
```

Comment:
@adamjstewart @tgamblin Given the discussion above, should I change this line back to `if '+mpi' in spec:`? I agree that starting to standardize style in packages would be helpful; frequently, I pattern-match idioms from other Spack packages when I write or modify a Spack package.